examples/bar.jl

```diff
@@ -35,38 +34,32 @@ function bar4()
     data = bar(;x = ["Product A", "Product B", "Product C"],
                 y = [20, 14, 23],
                 text = ["27% market share","24% market share", "19% market share"],
-                marker_color = "rgb(158,202,225)",
-                marker_opacity = 0.6,
-                line_color = "rgb(8,48,107)",
-                line_width = 1.5)
+				marker = attr(color = "rgb(158,202,225)", opacity = 0.6),
```

Comment:
hmm... that's really odd that github is showing that. Everything looks normal in my vim